Dockerfile

```diff
@@ -6,20 +6,28 @@ ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONIOENCODING 'utf-8'
 ENV LANG 'C.UTF-8'
 
-
-
 # text extraction stuff
-RUN apt update && apt install -y --no-install-recommends \
+# install git for forked dependency re: scrapelib
+RUN apt-get update -qq && apt-get install -y -qq --no-install-recommends \
```

Comment:
Just some clarification is the switch from apt to apt-get because of the forked dependency: scrapelib? Looks good though, ran this locally and is working as desired.

Reply:
`apt` isn't meant to be used in scripts, according to docs:
```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```
So kinda...standard best practice in docker files is to use `apt-get` instead.

Reply:
That makes sense. Thanks for clarifying that, good reason to switch it.